vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -135,20 +134,19 @@ In the top-right corner of the dashboard, you'll notice a toggle to switch betwe
 
 Next, you'll add a button that lets users export the data currently shown in your dashboard.
 
-Vizro provides several built-in actions you can reuse. You can explore the full list in our [guide on built-in actions](../user-guides/actions.md#built-in-actions).
+Vizro provides several built-in actions you can reuse. You can explore the full list in our [guide on built-in actions](../user-guides/actions.md).
```

Comment:
I think some readers would be saying to themselves, "🤔 What have actions got to do with exporting the data?" so can I suggest another sentence that bridges the gap between adding the button and linking to built-in actions. Something along the lines:

```suggestion
This is an example of an action, a Python function that is _triggered_ when a user interacts with a component, built into Vizro.
```